app/services/strategy_marketplace_service.py

```
@@ -675,35 +692,137 @@ async def get_user_strategy_portfolio(self, user_id: str) -> Dict[str, Any]:
+                        
+                        strategy_portfolio.append({
+                            "strategy_id": strategy_id,
+                            "name": f"Community Strategy {strategy_id}",
+                            "category": "Community",
+                            "monthly_cost": monthly_cost,
                             "performance": performance,
-                            "is_ai_strategy": True
+                            "is_ai_strategy": False,
+                            "tier": "community"
                         })
+                    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)